src/misc/msg.ts

```diff
@@ -0,0 +1,11 @@
+import { publish } from "../pubSub/pubSub";
+import { stdToGeneratorPublishedMessage as prefix } from "./helpers";
+
+/**
+ * Displays a message to the user
+ * @param {string} message The message to be displayed
+ * @param {string} mode Optional parameter. There are two modes to display the message: `nowait` and `status`
```

Comment:
How about defining an enum for the `mode`?

Reply:
I thought about it, but then the generator should map the string values to the corresponding enum, for this specific parameter in this specific function. Even though it would be better for the standard classes, I don't think the added complexity to the generator is worth it.